From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rahul Bedarkar <rabedarkar@nvidia.com>
Date: Thu, 3 Nov 2022 11:16:28 +0000
Subject: [PATCH 060/240] NVIDIA: SAUCE: tegra-epl: Remove dependency on
 ARCH_TEGRA_234_SOC

Remove dependency on ARCH_TEGRA_234_SOC config symbol. DT compatible
string can handle required dependency.

Bug 3847981

Change-Id: Iff984ccf9319a9330f934f0b506ade1a831ce112
Signed-off-by: Rahul Bedarkar <rabedarkar@nvidia.com>
Reviewed-by: Laxman Dewangan <ldewangan@nvidia.com>
Reviewed-on: https://git-master.nvidia.com/r/c/linux-stable/+/2902877
Tested-by: Abhilash G <abhilashg@nvidia.com>
Reviewed-by: Abhilash G <abhilashg@nvidia.com>

diff --git a/drivers/platform/tegra/Kconfig b/drivers/platform/tegra/Kconfig
index d8f93678d183..404281dc1e6e 100644
--- a/drivers/platform/tegra/Kconfig
+++ b/drivers/platform/tegra/Kconfig
@@ -17,7 +17,7 @@ if TEGRA_PLATFORM_DEVICES
 
 config TEGRA_EPL
 	bool "Tegra Error Propagation Layer Driver"
-	depends on ARCH_TEGRA_234_SOC && MAILBOX
+	depends on MAILBOX
 	help
 	  The tegra-epl driver provides interface for reporting software detected
 	  errors to Functional Safety Island (FSI) via HSP mailbox or MISC EC
-- 
2.55.0

